module-4/backend/week_two.md

```diff
@@ -1,22 +1,51 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's one difference between ES5 and ES6?
-2. What's the difference between asynchronous and synchronous JavaScript? 
+> Generally speaking, ES5 and ES6 use different syntax and have some differences in available functions.  To provide one example, ES5 uses `var` to declare variables where ES6 have the options of `let` and `const`.
+
+2. What's the difference between asynchronous and synchronous JavaScript?
+> Synchronous JS one line at a time where each line must complete before the next line start. Asynchronous JS can 'kick off'/execute code in parallel
```

Comment:
Yep! And make sure you're clear that JS is a synchronous language; we just have some functions that operate in an async manner.